newIDE/app/src/MainFrame/index.js

```diff
@@ -1235,6 +1233,11 @@ const MainFrame = (props: Props) => {
     storageProviders: props.storageProviders,
   });
 
+  const gamesPlatformFrameTools = UseGamesPlatformFrame({
+    fetchAndOpenNewProjectSetupDialogForExample,
+    onOpenProfileDialog: () => openProfileDialog(true),
```

Comment:
useCallback the onOpenProfileDialog, otherwise this means that `handleIframeMessage` is unstable and will be disconnected/reconnected at every single render 😱

This would probably be worth a log in the code doing the `addEventListener` so we can see the console filling with logs if we break the stability in the future.